src/tutorials/0005-regular-files-api/03.md

```diff
@@ -19,33 +19,52 @@ When you add a file to IPFS, you're putting it only in your own node, but making
 Let's take a look at how to add a file your IPFS node. We'll do this by executing the `add` method:
 
 ```javascript
-await ipfs.add(data, [options])
+ipfs.add(data, [options])
 ```
 
 So if we had the `File` object for an adorable photo of a kitten in our browser, accessible to us via a variable `catPic`, and we wanted to add it to our IPFS node, we could pass it into the `add` method as `data` like so:
 
 ```javascript
-await ipfs.add(catPic)
+ipfs.add(catPic)
 ```
 
 Note that the `add` method can accept either a single `File` or an array, just in case you have more than one adorable animal photo to add to the node:
 
 ```javascript
-await ipfs.add([catPic, dogPic, giraffePic])
+ipfs.add([catPic, dogPic, giraffePic])
 ```
 
-Because the `add` method returns a `Promise`, you'll need to place an `await` before the method call to suspend the execution until the return value of the promise is ready to be used.
+Because the `add` method returns an [`Async Iterable`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of), you'll need to call the iterable multiple times until you have all the results.
```

Comment:
```suggestion
Because the `add` method returns an [`Async Iterable`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of), you'll need to collect all the results by iterating over it until it is done.
```